File: Java/Day2.java
package Daywise;

public class Day2 {
    public static void main(String[] args){
        String paragraph = "Bob hit a ball";
        char [][]words = new char [10][10];
        int wordcount=0;
        for(int i=0,j=0,index=0;i<=paragraph.length();i++){
            //I am getting index out of bound exception

            while(paragraph.charAt(i)!=' ' && i<paragraph.length()){
                words[index][j]=paragraph.charAt(i);
                i=i+1;
                j=j+1;
            }
            wordcount=wordcount+1;
            words[index][j]='\0';
            index=index+1;
            j=0;
        }
        for(int i=0;i<wordcount;i++){
            int j=0;
            while(words[i][j]!='\0'){
                System.out.print(words[i][j]);
                j++;
            }
            System.out.println();
        }
    }
}

File: CLion/Day2.cpp
#include<stdio.h>
int main(void) {
    char paragraph[100] = "Bob hit a ball Bob hit a ball Bob hit a ball Bob hit a ball";
    char words[10][10];
    int wordCount=0;
    for (int i=0,j=0,index=0;i<sizeof(paragraph)-1;i++) {
        while (paragraph[i]!=32 && i<sizeof(paragraph)) {
            words[index][j]=paragraph[i];
            i=i+1;
            j=j+1;
        }
        wordCount=wordCount+1;
        words[index][j]='\0';
        index=index+1;
        j=0;
    }
    for (int i=0;i<wordCount;i++) {
        int j=0;
        while (words[i][j]!='\0') {
            printf("%c",words[i][j]);
            j++;
        }
        printf("\n");
    }
    return 0;
}